polaris.shopify.com/src/data/icons.json

```diff
@@ -3,8 +3,14 @@
     "name": "abandoned cart",
     "set": "major",
```

Comment:
My script does exactly the same thing, we should remove mine. If the script runs in `polaris.shopify.com`, the script needs to be manually run to update the icons page which is what you wanted to avoid. It would be better if polaris-icons handled all the logic and exposed the meta json. (Riiight?)